README.md

```diff
@@ -432,7 +392,7 @@ class RethinkDBFSBucket {
   /**
    * Find and return the files table documents that match @filter.
    */
-  Iterable find(Document filter, RethinkDBFSFindOptions options=null);
+  Iterable find(Document filter, RethinkDBFSFindOptions=null);
 
```

Comment:
right, are those necessary? From a JavaScript standpoint (the world I live in). Its redundant.